_common/pgadmin/servers.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Servers": {
+    "1": {
+      "Name": "TimescaleDB",
+      "Host": "postgresql",
+      "Group": "Acra - TimescaleDB example",
```

Comment:
Hm, is it only related to `TimescaleDB`, if I am not mistaken, pgAdmin image is using almost in all demos we have

Reply:
yeah, you are right. forgot about that. will change to some common `Acra - Postgresql example`.